docs/source/plugin/what.rst

```diff
@@ -0,0 +1,133 @@
+=================
+What is a Plugin?
+=================
+
+Before writing your first plugin, you may want to know what is a plugin, how is
+it made, and what are the best solutions for your need. In this chapter we'll
+cover what is a Plugin, what you can do with them, and some vocabulary.
+
+You may want to skip ahead to :doc:`anatomy` if you already know what a Plugin
```

Comment:
```suggestion
You may want to skip ahead to :doc:`anatomy` if you already know what a plugin
```

I'm going on a crusade against these unnecessarily capitalized words, if you couldn't tell. 😹

Reply:
Unlike the ones in Human History, this is a rightful one.